pkg/kapp/diff/change_set_with_versioned_rs.go

```diff
@@ -33,7 +33,7 @@ func NewChangeSetWithVersionedRs(existingRs, newRs []ctlres.Resource,
 
 func (d ChangeSetWithVersionedRs) Calculate() ([]Change, error) {
 	existingRs := existingVersionedResources(d.existingRs)
-	existingRsGrouped := d.groupResources(existingRs.Versioned)
+	existingRsGrouped := newGroupedVersionedResources(existingRs.Versioned)
```

Comment:
Should we also rename this variable to `existingVersionedRsGrouped`? Just a though not a blocker though.